src/libs/balance-set/src/balance_set.nr

```diff
@@ -26,8 +38,10 @@ impl BalanceSet<UtilityContext> {
 
     pub unconstrained fn balance_of_with_offset(self: Self, offset: u32) -> u128 {
         let mut balance = 0 as u128;
+        // docs:start:view_notes
         let mut options = NoteViewerOptions::new();
         let notes = self.set.view_notes(options.set_offset(offset));
+        // docs:end:view_notes
```

Comment:
watch out here, should we already import their balance_set so that we don't need to update the files on each upgrade?